project/Dependencies.scala

```diff
@@ -26,14 +26,16 @@ object Dependencies {
   val play25Ws            = "com.typesafe.play"          %%    "play-ws"                  % play25Version  % "test"
   val play25Test          = "com.typesafe.play"          %%    "play-test"                % play25Version  % "test"
 
-  val play27WsJson        = "com.typesafe.play"          %%    "play-ws-standalone-json"  % "2.0.4"  % "test"
-  val play27Json          = "com.typesafe.play"          %%    "play-json"                % "2.7.1"
+  val play27WsJson        = "com.typesafe.play"          %%    "play-ws-standalone-json"  % "2.0.6"  % "test"
+  val play27Json          = "com.typesafe.play"          %%    "play-json"                % "2.7.4"
   val play27              = "com.typesafe.play"          %%    "play"                     % play27Version
   val play27NettyServer   = "com.typesafe.play"          %%    "play-netty-server"        % play27Version  % "test"
   val scalaTestPlusPlay27 = "org.scalatestplus.play"     %%    "scalatestplus-play"       % "4.0.1"  % "test"
 
+  val play27NettyUnix     = "io.netty" % "netty-transport-native-unix-common" % "4.1.34.Final" % "test"
+
   val testDependencies = Seq(scalaCheck)
-  val play27Dependencies = Seq(refined, play27, play27Json, play27NettyServer, scalaTestPlusPlay27, play27WsJson)
+  val play27Dependencies = Seq(refined, play27, play27Json, play27NettyServer, scalaTestPlusPlay27, play27WsJson, play27NettyUnix)
```

Comment:
I have no idea why this is now necessary, the dependency should have been added transitively by `play-netty-server`.